test/libsolidity/smtCheckerTests/loops/while_bmc_iterations_semantics_2.sol

```diff
@@ -7,15 +7,14 @@ contract C {
 				y = 1;
 			++x;
 		}
-        // BMC loop iteration setting is just enough to leave the loop
-		assert(x == 3); // should hold
-		assert(y == 1); // should fail
+ 		// BMC loop iteration setting is just enough to leave the loop
+		assert(x == 3);
+		assert(y == 0);
```

Comment:
Why not keep the failing assertions? Just to know that the first one is not safe for the wrong reasons.

Reply:
Right!